src/crypto/tests/TestChipCryptoPAL.cpp

```diff
@@ -121,15 +119,15 @@ class HeapChecker
 class HeapChecker
 {
 public:
-    explicit HeapChecker(nlTestSuite *) {}
+    explicit HeapChecker() {}
```

Comment:
It seems we need to change above line 95 as well.
Do we never use MBEDTLS_MEMORY_DEBUG in CI?

Reply:
Fixed